### ISSUE TITLE
IPv6 support

### ISSUE THREAD

**Issue by HikariWS** [lstein/Net-ISP-Balance#28]
Hello. How is Net-ISP-Balance support for IPv6, including global prefix delegation?

Currently I have a Cisco RV340 load balancing 2 ISPs.

One modem I set as bridge and another I left in router mode. The bridged WAN is able to receive and pass global prefix using DHCP-PD to VLAN, whose DHCP server distributes. The other ISP (in route mode)'s WAN has DHCP-PD set, but VLAN's DHCP server says it's unaccessible.

Load balancing works nicely for IPv4 and even speedtest reports combined download speed. But for IPv6 my LAN devices receive only addresses from bridged ISP's global prefix and only use its bandwidth.

RV340 also seems to not support setting ULA together with DHCP-PD, as I've seen ppl using in other routers. This makes me unable to set fixed IPv6 addresses on LAN, as everytime ISP changes its global prefix I get all devices addresses changed too.

To sum things up, I use Pi-hole for local domains attribution, DNS server and DHCP server. I'm unable to make dnsmasq grab global prefix delegation from RV340 so it can distribute addresses for devices.

I'm then considering removing RV340 and setting up a Ubuntu or IPFire gateway, which would load balance both ISPs and handle DNS, DHCPv4 and DHCPv6 all together.

If anybody has done something like that or trying to, feel free to contact me and share configs and tools.

**Comment by rp2019433** [comment]
Hi.

Good question. Any plans to support IPv6 on this tool?

Btw, thanks for this excellent project.

**Comment by lstein** [comment]
I do not currently have plans to add IPv6 support. My local ISPs do not provide such support and I have no way of testing it.
